spec/features/admin/admin_sees_order_dashboard_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.